src/pages/Survey.tsx

```diff
@@ -127,42 +130,62 @@ export const Survey = () => {
     }
   }, [data, loading, error, toast]);
   return (
-    <Flex
-      w="100%"
-      minH="calc(100vh - 5.5rem)"
-      pt={30}
-      alignItems="center"
-      justifyContent="center"
-      flexDirection="column"
-    >
-      <Container mb={32} maxW="container.lg">
-        <Heading mb={6}>Professor Preferences Survey</Heading>
+    <>
+      {user != null && user.preferences.length > 0 ? (
+        <>
+          {" "}
```

Comment:
You can remove this {" "}, Sometimes if you have an extra space somewhere prettier puts this in, in this case I don't believe it changes anything

Reply:
good catch!